Add overview page to advanced tutorials. (#304)
micro-ROS/micro-ROS.github.io: _docs/tutorials/advanced/overview/index.md

---
title: Overview
permalink: /docs/tutorials/advanced/overview/
redirect_from:
  - /docs/tutorials/advanced/
---

> This chapter provides a number of advanced tutorials for those users who already have some previous micro-ROS knowledge. They are useful to interact with micro-ROS at a deeper level if compared with the [**First Step Tutorials**](../../core/overview). There is no specific order recommended to take these tutorials, as each addresses a different aspect of the micro-ROS stack and toolchain.

Reviewer: ```suggestion
This chapter provides a number of advanced tutorials for those users who already have some previous micro-ROS knowledge. They are useful to interact with micro-ROS at a deeper level compared with the [**First Step Tutorials**](../../core/overview). There is no specific order recommended to take these tutorials, as each addresses a different aspect of the micro-ROS stack and toolchain.
```

* [**Optimizing the Middleware Configuration**](../microxrcedds_rmw_configuration/)

>   In this tutorial, we'll guide you through the configuration of the middleware between a microcontroller and the micro-ROS agent running on some Linux-based microprocessor, to optimize it for your specific use-case an application.

Reviewer: ```suggestion
  In this tutorial, we'll guide you through the configuration of the middleware between a microcontroller and the micro-ROS agent running on some Linux-based microprocessor, to optimize it for your specific use-case and application.
```

* [**How to include a custom ROS message in micro-ROS**](../create_new_type/)

>   This tutorial explains how to create a custom ROS message type to micro-ROS, and then add it to the [build system](https://github.com/micro-ROS/micro_ros_setup).

Reviewer: ```suggestion
  This tutorial explains how to create or include a custom ROS message type in a micro-ROS application - and in particular how to bring it into the [build system](https://github.com/micro-ROS/micro_ros_setup).
```
